backend/users/tests/test_user.py

```diff
@@ -102,22 +103,23 @@ def test_retrieve_user(self):
         response = AuthenticatedClient(channel=self.channel).get(url, **self.extra)
         assert response.status_code == status.HTTP_200_OK
 
+    @parameterized.expand(
+        [
+            ({"role": UserChannel.Role.USER},),
+            ({"password": "newpas$w0rdtestofdrea6S43"},),
+            ({"ui_preferences": {"columns": ["col1", "col2"]}},),
```

Comment:
Could you add a test on the fact that ui_preferences is partially updated and not fully replaced? That is keys that were not in the request payload are left untouched.

Reply:
Test added.